pages/glossary/focusable.md

```diff
@@ -8,4 +8,15 @@ input_aspects:
   - DOM tree
 ---
 
-Elements that can become the target of keyboard input as described in the [HTML](https://www.w3.org/TR/html) specification of [focusable](https://html.spec.whatwg.org/#focusable-area) and [can be focused](https://html.spec.whatwg.org/#specially-focusable).
+An element is _focusable_ if one or both of the following are true:
+
+- the element is part of [sequential focus navigation][]; or
+- the element has a [tabindex value][] that is not null.
+
+**Exception**: Elements that lose focus immediately after the focus event is fired are not considered _focusable_.
+
+The [tabindex value][] of an element is the value of the [tabindex attribute][] parsed using the [rules for parsing integers](https://html.spec.whatwg.org/#rules-for-parsing-integers). For the [tabindex value][] to be different from null, the [tabindex attribute][] must be a [valid integer](https://html.spec.whatwg.org/#valid-integer).
```

Comment:
I actually ran into an issue where a site was using `tabindex="-1-"` (extra `-` at the end), which doesn't parse as a valid integer according to HTML spec, and therefore Alfa was rejecting it, but Chrome doesn't care and treated it as `tabindex="-1"`.
We should at least have some note about that. It seems to only create false negative for now (something would be accepted by the rule due to having an incorrect tabindex, even though it would be an actual problem in Chrome (and maybe others)), so we maybe don't want to do more than a note…

Reply:
Interesting! Didn't think that could work. Perhaps a little more research here is needed. Figure out exactly what will and will not be accepted.

Reply:
`tabindex="-1-"` is parsed by the algorithm as `-1` (so, not only Chrome, but all browsers respect the spec)

I've updated the last sentence to be correct, since it wasn't.

Reply:
🤔 Ah, that's actually more twisted (as always…)
The definition of [valid integer](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-integer) is
> A string is a valid integer if it consists of one or more ASCII digits, optionally prefixed with a U+002D HYPHEN-MINUS character (-).

(so `-1-` is not a valid integer)

…

but the [rules for parsing it, point 8](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#rules-for-parsing-integers) just below states:
> Collect a sequence of code points that are ASCII digits from input given position

thus disregarding anything after the first non-digit included (so `123etuirentiuernautier456` parses as `123`)…